libecho/tests/packet_tests.cpp

```diff
@@ -0,0 +1,266 @@
+#include <gtest/gtest.h>
+#include <algorithm>
+#include <cassert>
+#include <cstdint>
+#include <string>
+#include <vector>
+
+#ifdef _WIN32
+#include <winsock2.h>
+#elif defined __unix__
+#include <arpa/inet.h>
+#endif
+
+#include "CRC.h"
+#include "Packet.h"
+
+uint16_t SIZE = 3, NUM = 2137;
+const std::vector<Flag> SET_FLAGS{Flag::ACK, Flag::DMD, Flag::LPC}, UNSET_FLAGS{Flag::SYN, Flag::FIN, Flag::RST};
+const std::vector<uint8_t> header{0, 38, 0, 3, 8, 89}, data{13, 4, 55}, incorrect_header{10, 37, 0, 4, 1, 2, 70},
+    incorrect_data_small{1}, incorrect_data_big{12, 34, 56, 78}, incorrect_crc_format_small{12},
+    incorrect_crc_format_big{12, 13, 14, 16, 17}, incorrect_crc{1, 1, 1, 1};
```

Comment:
I think it would be much more readable, if every variable was in distinct line.
```suggestion
const std::vector<uint8_t> header{0, 38, 0, 3, 8, 89},
                           data{13, 4, 55},
                           incorrect_header{10, 37, 0, 4, 1, 2, 70},
                           incorrect_data_small{1},
                           incorrect_data_big{12, 34, 56, 78},
                           incorrect_crc_format_small{12},
                           incorrect_crc_format_big{12, 13, 14, 16, 17},
                           incorrect_crc{1, 1, 1, 1};
```